internal/util/util.go

```diff
@@ -134,14 +134,14 @@ func GetVolumeSnapshotClassForStorageClass(provisioner string, snapshotClient sn
 	return nil, errors.Errorf("failed to get volumesnapshotclass for provisioner %s", provisioner)
 }
 
+// GetVolumeSnapshotContentForVolumeSnapshot returns the volumesnapshotcontent object associated with the volumesnapshot
```

Comment:
Minor nit: Period at the end for valid godoc string.

Reply:
added the `.` in https://github.com/vmware-tanzu/velero-plugin-for-csi/pull/37/commits/e4018c28326f338f6c346f6e8d40502d8c66ae7b